contracts/xasset-orchestrator/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+#![no_std]
+
+use loam_sdk::derive_contract;
+use loam_sdk::soroban_sdk::{Address, Symbol, String};
+use loam_subcontract_core::{admin::Admin, Core};
+
+pub mod error;
+use crate::error::Error;
+pub mod orchestrator;
+pub use orchestrator::*;
+
+pub mod data_feed {
+    use loam_sdk::soroban_sdk;
+
+    soroban_sdk::contractimport!(
+        file = "../../target/wasm32-unknown-unknown/release/data_feed.wasm"
+    );
+}
```

Comment:
This should work with `import_contract` just need to have data_feed as a dep so that that will be built first with `loam build`